front-end/src/app/app.component.ts

```diff
@@ -1,35 +1,126 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, inject } from '@angular/core';
 import { Task } from 'src/model/Task';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { TaskCardComponent } from './task-card/task-card.component';
+import { TaskService } from './task.service';
 
 @Component({
   selector: 'app-root',
-  templateUrl: './app.component.html',
+  template: `
+    <div class="container dashboard">
+      <form [formGroup]="applyForm" (submit)="submitTask()">
+        <mat-form-field class="task-form-field">
+          <mat-label>New Task</mat-label>
+          <input matInput type="text" #task_title formControlName="title" />
+        </mat-form-field>
+        <mat-form-field class="task-form-field">
+          <mat-label>Comment</mat-label>
+          <textarea
+            matInput
+            type="text"
+            #task_comment
+            formControlName="comment"
+          ></textarea>
+        </mat-form-field>
+
+        <button
+          mat-raised-button
+          (click)="task_title.value = ''"
+          (click)="task_comment.value = ''"
+          color="primary"
+          class="add_btn"
+          style="display: block; padding: 10px"
+          type="submit"
+        >
+          Add Task
+        </button>
+      </form>
+      <div class="row">
+        <div class="col-sm-5 task-container" *ngIf="gotTaskArray">
+          <h3 style="color: #3f51b5; text-decoration: underline">Tasks:</h3>
+          <app-task-card
+            *ngFor="let task of taskArray"
+            [task]="task"
+            [isDone]="true"
+            (doneEvent)="doneTask($event)"
+            (deleteEvent)="deleteTask($event)"
+          ></app-task-card>
+        </div>
+        <div class="col-1"></div>
+        <div class="col-sm-5 done-container" *ngIf="!doneArrayEmpty">
```

Comment:
itt van egy UX probléma: ha nagyon sok elem van a listában, akkor nagyon sokat kellene legörgetnem, hogy meg tudjam nyomni a `Clear` gombot. Több lehetőség is van ezt javítani:
1. a clear gombot a div tetjére teszed
2. beállítasz egy maximum magasságot, amit a div elfoglalhat, és ilyenkor görgethetővé teszed a listát

az utóbbi szofisztikáltabb, robosztusabb megoldás, mert pl ha lenne valamilyen content a listák alatt, akkor nem kéne az 1000 elemből álló done listát végig görgetnem